shared/chat/conversation/messages/cards/team-journey/index.tsx

```diff
@@ -93,7 +95,7 @@ const styles = Styles.styleSheetCreate(
     ({
       actionsBox: {
         marginTop: Styles.globalMargins.tiny,
-        minHeight: 50,
+        marginBottom: 22,
```

Comment:
This was done to make the wave button in its "wave completed" state line up with the other button. Would this mess up chat message height calculations?